### ISSUE TITLE
Fail to add 2 items with the same property name with dat.GUI style

### ISSUE THREAD

**Issue by edankwan** [wearekuva/oui#10]
For example:
```js
    var gui = oui.datoui();
    oui.annotate({label: 'foo'})(gui.add( uniforms.u_foo, 'value'), 'value');
    oui.annotate({label: 'bar'})(gui.add( uniforms.u_bar, 'value'), 'value');
```

**Comment by marklundin** [comment]
Yep, currently values are indexed using the propName hence the collisions. These should be indexed numerically in the oder they're added

**Comment by marklundin** [comment]
Fixed in 0.0.14
